Ghidra/Processors/SuperH4/data/languages/SuperH4.sinc

```diff
@@ -1486,6 +1486,56 @@ N_0txx: r0",@"^N_0  is r0 & N_0 { tmp:4 = N_0; export tmp; }
 	<skip>
 }
 
+# Floating Point Square Reciprocal Approximate
+# pattern 1111nnnn01111101			
+# text	  fsrra <F_REG_N>
+# arch    # arch_sh2a_or_sh3e_up
+:fsrra FRN_0t	is
+	OP_0=0xF & FRN_0t & DRN_1t & OP_4=0x7D {
+	if (!( $(FPSCR_PR) == 0 )) goto <doublePrecision>;
```

Comment:
just assign single precision and then if FPSCR_PR assign again as double  or `goto inst_next` (ignorantly I'm assuming its the same register space, so may not be 100% applicable if that's that not accurate, but probably solutions exist with fewer internal gotos)

Reply:
Looks like it is undefined behaviour anyways if FPSCR_PR is not 0.